packages/README.md

```diff
@@ -0,0 +1,14 @@
+# Cal-ITP Python Packages
+
+We publish calitp-data-analysis and calitp-data-infra, two Python packages that enable work elsewhere in the repository. Each is maintained and deployed the same way.
```

Comment:
Could maybe link to each of their respective PyPI pages? Just to kind of emphasize that these are truly published packages

Reply:
Addressed in [ac52c02](https://github.com/cal-itp/data-infra/pull/3156/commits/ac52c0287d433386871beca0c45af14ede6b1f84)